components/Styled.js

```diff
@@ -0,0 +1,117 @@
+// import styled from 'styled-components/native'
```

Comment:
How come you didn't use this file?
Looks like a good idea to extract all the styling into it's own file and then import the styled components where you want them?
just remember the export
`export const Container = styled.View`

Reply:
if not, maybe delete this file instead.